docs/detections/alerts-ui-manage.asciidoc

```diff
@@ -1,27 +1,74 @@
+* Augment the results shown in the Alerts table to include alerts from building blocks or only show alerts from indicator match rules. By default, building block alerts are excluded from the Alerts table. Opting to include them expands the results and displays building block alerts alongside regular detection alerts.
+
+NOTE: When updating alert results to also include building block alerts, the Security app searches the `.siem-signals-<Kibana space>` index for the `signal.rule.building_block_type` field. When looking for alerts created from indicator match rules, the app searches the same index for the `signal.rule.threat_mapping` field.
+
+[role="screenshot"]
+image::images/additional-filters.png[Shows multiple ways to filter information]
+
+[float]
+[[customize-the-alerts-table]]
+=== Customize the Alerts table
+Use the icons in the upper left corner of the Alerts table to customize displayed columns and to view the table in full screen mode.
+
+[role="screenshot"]
+image::images/alert-table-columns-and-size.gif[width=100%][height=100%][Demo that shows how to select the customize display icon and full screen icon]
+
+Click the *Customize Event Renderers* icon to enable event renderers within the Alerts table. When enabled, event renderers show relevant details that provide more context to the event. For example, if you enable the *Flow* Event Renderer, the Alerts table shows relevant details describing the flow of the data between a source and destination. These details could include hosts, ports, protocol, direction, duration, amount transferred, process, and geographic location.
+
+[role="screenshot"]
+image::images/customize-event-renderer.png[Shows the Event Renderer icon, 200]
+
+All event renderers are disabled by default. To pivot between event views in the Alerts table, you can enable individual event renderers or click *Enable all*. Closing *Customize Event Renderers* page saves your configurations.
+
+[role="screenshot"]
+image::images/customize-event-renderer-page.png[Shows the Event Renderer page]
+
+[float]
+[[view-alert-details]]
+=== View alert details
+To inspect an alert further, click the *View details* icon from the Alerts table.
+
+[role="screenshot"]
+image::images/view-alert-details.png[Shows the Event Renderer icon, 200]
+
+The Alert details flyout appears and offers several options for viewing alert details:
+
+* *Summary*: Shows details about the alert. If the alert has been enriched with indicator match fields, the *threat summary* section also displays on the *Summary* tab and provides the details for the following fields:
+** `matched.field`
+** `matched.type`
+** `source (threat.indicator.provider)``
+** `first_seen`
+** `last_seen`
+* *Threat Intel*: Shows the number of threat intelligence sources. If you you haven't enabled any, this tab displays the messaage `No Threat Intel Enrichment Found`.
```

Comment:
Typo: 
```suggestion
* *Threat Intel*: Shows the number of threat intelligence sources. If you haven't enabled any, this tab displays the messaage `No Threat Intel Enrichment Found`.
```

Reply:
I think it may be more accurate to describe these as "indicator enrichments" or "matched indicators," or something to that effect; "intelligence sources" conveys (to me) a broader idea than what we're displaying on this tab. I don't know if we have standardized this language.

@MikePaquette @shimonmodi any thoughts here? The [Enrichment RFC](https://github.com/elastic/ecs/pull/1400) is currently describing these fields generally as "threat enrichments", of which "indicator rule match" may be a subtype.

Reply:
@rylnd can you clarify what is actually displayed in that location?

Reply:
what we are displaying is each item within the threat.indicator array on that specific alert. So we are displaying information copied over from the indicator document for each threat match on an alert. Chatted with Nastasha this morning and we agree with Ryland's comment, we will be updating this piece

Reply:
Thanks @ecezalp  SGTM